static/scripts/setup/system-check.sh

```diff
@@ -6,62 +6,101 @@ if hash ibmcloud 2>/dev/null
 then
     printf '\xE2\x9C\x85 IBM Cloud CLI \n'
 else
-    printf '\xE2\x9D\x8C IBM Cloud CLI \n'
+    printf '\xE2\x9D\x8C IBM Cloud CLI \n \n'
+
+    printf 'Download the IBM Cloud CLI using the links below: \n'
+    printf 'For All Users: https://cloud.ibm.com/docs/cli/reference/ibmcloud?topic=cloud-cli-install-ibmcloud-cli \n \n '
 fi
 
 if hash git 2>/dev/null
 then
     printf '\xE2\x9C\x85 Git CLI \n'
 else
-    printf '\xE2\x9D\x8C Git CLI \n'
+    printf '\xE2\x9D\x8C Git CLI \n \n'
+
+    printf 'Download the Git CLI using the links below: \n'
+    printf 'For All Users: https://git-scm.com/book/en/v2/Getting-Started-Installing-Git \n \n'
 fi
 
 if hash crc 2>/dev/null
 then
     printf '\xE2\x9C\x85 CRC \n'
 else
-    printf '\xE2\x9D\x8C CRC \n'
+    printf '\xE2\x9D\x8C CRC \n \n'
+
+    printf '* A RedHat Account is Required * \n'
+    printf 'Download CRC using the links below: \n'
+    printf 'For All Users: https://cloud.redhat.com/openshift/install/ \n \n'
```

Comment:
This is a more direct link to crc https://cloud.redhat.com/openshift/install/crc/installer-provisioned